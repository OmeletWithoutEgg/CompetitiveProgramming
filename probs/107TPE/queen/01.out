635.00 -5371.00
