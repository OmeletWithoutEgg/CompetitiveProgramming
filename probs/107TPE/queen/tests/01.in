938
-37877 66216
89834 59603
91098 -33207
-52892 755
37359 -63363
-79470 85761
43086 1207
-1362 58011
34771 88502
61318 -59395
58085 -47722
55116 58501
15513 -43235
59661 -65533
-63337 81634
-73504 79783
-88358 -55717
-80345 53939
47252 -18266
86406 -84922
18141 96614
-17659 31030
59653 -74663
-51481 -66828
-66480 -64679
64868 -17475
24649 -77192
-5112 -38298
72692 5339
-82945 27183
68792 77232
13665 -23411
97993 -36524
-54504 -3497
-14618 -42563
68565 99948
-15474 -40120
-1364 -12824
51239 12549
22624 35821
27164 79373
-73003 -17763
-24336 -42508
-56040 55357
-81615 -14490
-17142 82817
74476 -20587
-60669 -45337
46403 62863
29302 733
53021 -25379
30840 83287
-94075 87523
50841 -66165
-95363 -78543
8935 1584
-43690 81393
55340 67650
-93966 -47194
72413 -38982
-36205 32422
-94797 -43844
93346 -15882
-60569 44086
-39634 -67044
-69689 29015
13155 71019
-68128 32063
44762 47633
50878 -49972
72375 -10847
53636 -6190
81431 -98896
78047 -37265
84768 -93363
-86272 -959
45928 -83913
-53435 -40537
10230 -7750
749 10785
25962 8009
-83375 -4593
-93856 -34139
8254 74156
14135 77763
6060 97240
-80037 -89703
55473 -26341
-52160 59220
-94183 37785
-26716 -42413
62782 7761
61811 -90502
-55148 -19173
-94122 57465
-21176 60611
5971 83456
79803 -46438
-36024 79968
74346 -67051
19769 57776
33133 16035
44991 64337
-26913 81429
-36046 72506
9743 -72573
42874 45684
15595 33757
-1470 -12526
-42946 -29858
32198 -78809
67005 47899
58158 -48749
96063 -84014
-32468 39316
44667 -40727
84559 63505
-6749 4644
10511 5155
74258 -13722
-17774 -88931
-24352 25251
7639 78048
-92096 99962
-89566 76288
10032 -71830
-7446 -25997
-73416 23169
-51302 -22717
96232 17926
-61223 25550
44502 19377
66809 -12221
92459 -63707
89542 80263
-7863 54475
-17917 8319
94711 95928
-28307 76651
62840 -18727
26823 63423
-30236 -99693
-90656 -91398
-58700 36585
64804 -43855
94850 55215
-63427 3330
92407 66327
-81412 -50705
-95724 13985
90853 82756
77661 42314
80469 20571
-31394 -6148
50697 -43922
74892 7202
47789 49389
65006 -15990
84722 19966
33336 -96013
-14881 44660
-71892 -34459
80014 62967
-11662 -93866
-7403 -74389
-21335 56340
-66193 -49910
-13754 5264
3433 -94179
-55958 78074
24952 -99955
73397 -96547
-62348 92144
-80653 41940
-89635 26269
-96452 1045
-2893 52958
-46956 85649
-22352 -40759
-37490 23106
96351 88895
-26898 -72090
-77291 85109
-29418 -23147
-20622 -80419
74913 65740
39569 -83666
-40628 -9634
-90750 80696
-49649 -35053
-30052 -80974
10053 -83967
-21013 20188
42246 28241
40333 16145
72438 40874
84992 2782
93976 -55134
-76407 6967
85147 -9508
74545 -51596
-81364 5767
-6507 42015
15485 5654
-3642 38475
85059 46338
-49573 -72501
20711 -60776
15750 55262
-19787 -85798
-66740 96473
-8609 29774
25374 23300
87797 92750
-85330 -19309
48800 13699
98786 -96095
-74785 -77334
-63627 48945
1664 73751
41699 -76978
-90166 -94464
-63183 -55039
-71154 75205
-72980 -49631
19127 -97254
24140 -62552
29719 -3103
-6615 -13227
99396 -33765
67922 86431
20495 -14482
38979 -88247
-66106 96763
77411 20234
70280 65758
-15191 5639
-43350 -68943
-5452 -69823
-10447 80025
68024 45101
23935 -2251
60611 -35047
-21508 -25303
72404 -48015
86408 -45835
-13260 -90831
-55631 -98357
88574 29557
-5335 -30303
-17820 41819
85181 89090
-83030 57863
-42071 30794
18416 10341
-98323 -33101
-5520 91509
-44111 -10049
89922 -53069
85824 59793
-83626 38777
-1354 -8649
91956 31942
-13734 -74911
-7328 25273
-26934 13901
10016 -96589
-32739 -48462
-75149 -80963
41041 -82203
-65301 -42259
32337 25187
-80079 -66186
-37066 52938
8951 -49439
-82929 75035
38593 76203
-22351 87310
84699 52250
87211 -32680
20664 96011
-77614 11251
-18315 39894
-94187 -86643
77304 17382
5374 45069
-30100 -46156
-95230 56576
33035 -86580
88050 -78881
98434 -54556
-83103 48377
-12685 36594
-62710 88144
-23625 -92812
-77890 -27409
19951 -98142
13391 -98629
44184 -97477
74698 -7485
-27391 -48989
-45240 67732
-9850 -34660
-24196 -16782
25518 -60662
34526 59895
-79649 63529
88818 61974
82682 78348
17396 -76089
54227 22575
-46846 29553
45187 -91364
-24370 53670
-31929 63283
14088 38822
-59316 -85883
41833 34566
-12067 -90927
39709 -33726
74491 36879
70696 -81907
-83001 3343
-23638 -29424
30293 -37902
-27437 75053
-41005 -56980
-87594 72924
99835 29720
77261 96684
50693 86934
23865 -58599
31824 -64905
66785 -81283
-63981 87174
-13069 19772
66219 -98607
13731 -1167
1118 28292
-92673 59416
61919 -73860
73158 47072
-18143 76248
-50098 -77660
-63953 -10796
-12425 -21654
74499 27818
96927 14079
15031 54038
-20337 -42806
4021 -57259
-52670 -40631
30887 50047
-29910 21399
23885 -86131
29624 4175
-65778 18080
-83659 45562
34741 13767
-50423 27279
1432 30601
-98324 89707
-7837 9905
-60250 93453
-57092 -41269
62824 27042
21227 -37568
23840 -55677
25417 -30489
71846 -19110
17345 -91788
79644 76206
-94553 52162
28108 -21552
71640 -57388
39405 65796
-97307 76766
64643 95961
39908 -17416
-3341 -92681
-41790 99827
69451 78214
-12141 44494
36909 40556
89538 4922
87528 392
-13699 -44975
63072 -22638
-75092 -57116
-76622 79170
19044 93371
-28978 -36738
-14047 15513
-46588 -59219
-23522 -47783
29384 -80132
-91559 -60517
-30614 -64572
39193 74991
64052 4518
-42778 27518
-22514 -45377
-93707 -66274
-87340 44210
-23532 -2434
20856 -29833
29133 39534
55191 99047
10063 77795
48756 82703
-45993 -73688
-49559 -67105
-79033 -88124
86488 57273
68028 96441
81336 -12798
-65643 -2020
-59554 96260
84365 -4596
92048 -55541
-34083 -19391
-85439 81784
-619 89109
-20766 73864
-71764 -30673
44424 95031
89526 -23253
6503 32316
-39392 57170
-96025 -67541
-34935 90035
15488 -34138
-17526 58591
-12990 56381
-10194 -87129
-1305 -4528
-19352 -67058
-36446 -24387
-13582 83425
46314 65095
79090 13506
23216 99708
59059 56429
-10780 -52256
96026 74228
29304 46627
72282 98258
-82989 35577
95889 19484
66163 -84644
-95154 56975
-80152 35727
-41021 -19179
50016 -16484
73575 66613
-77049 -38855
-26676 40517
4426 -29459
-45624 28387
-16378 61711
-30418 -32230
95056 -41745
-90097 50199
89357 -61508
-80295 -58188
80267 6056
-80582 -16878
-79645 -53895
-82834 -8829
-11466 -94440
50566 43918
-47947 48818
-1395 -68957
60126 13835
-41563 57639
92232 -25776
-83477 28695
-83074 56724
95557 2627
-42316 772
31423 -16071
-45039 10463
80163 19530
-4005 85598
36650 -8300
72627 40892
-10323 2855
-37092 43417
48030 -12375
70601 -921
-86920 -51282
16307 -11557
-61569 -46165
41817 -2340
-90768 -687
-31211 7016
892 96098
80617 -95863
-97686 76108
47137 99789
-90476 79581
-90003 11070
52099 -76432
50463 93266
42779 24109
-80687 -63003
-83963 65179
-143 -789
3304 81214
22439 -46163
-15081 -91083
-72292 -23405
-21295 34587
89674 -83395
-90957 -52352
15461 21358
27531 -71415
-42040 85930
36350 -45251
45919 -42792
-15307 60791
-41206 -53743
89423 97480
26 -25950
40417 -4233
26524 -22327
94696 -21273
75569 -10461
-60965 -39843
52508 11062
-93242 69281
-10992 74059
-32870 -70700
72346 74354
54609 -87369
-10464 -48654
33353 -73949
29523 77704
-95482 -65067
84526 10792
615 -66958
-23635 -89693
-1792 85713
-60677 12530
-30254 19996
-88493 12718
28183 61281
-50882 97525
4492 -64570
-94508 -94030
97335 49249
61431 -58814
72661 -39066
-11783 -52911
-1710 -90300
-11495 -69629
87279 -76115
29248 -86942
-59519 14354
59335 -46872
-79404 65127
94039 -62769
-79201 88194
-29647 -53398
-1313 -61072
68485 3336
32600 36267
-81414 -86715
8191 -83140
-94138 -87204
-82466 -20908
1058 -5461
20393 -73536
-91828 -44207
-59066 -18902
86903 1423
55621 -57385
97358 -10312
43044 -21809
-46667 -11355
-35205 -98913
-71826 10302
84841 49678
-92718 -78096
40503 -61807
74395 60800
80429 33752
-55024 5524
-42077 4406
8072 -54238
-20303 -43145
22273 -55353
81705 -53180
-17210 10883
85946 -53855
95429 82528
-75374 57857
98806 34773
26818 -44243
-70830 -96763
8758 64118
53370 93214
54688 81872
-57460 -65691
8197 -54296
-34662 -9128
-87758 -36046
-60867 27283
-35849 29888
78289 14502
-6440 -87039
-13193 -4999
70694 19319
-88462 10468
75364 88120
-54878 -2610
43572 20776
-92344 17502
21662 72713
-508 -54050
75839 -831
-13872 75776
-37305 27615
6361 -89519
-87750 -44009
-69473 92145
62528 39441
99061 25717
-36226 25064
-48055 -33143
-40226 -19484
60102 99796
37939 -89517
30983 7532
-35576 31093
-52203 -6216
11608 -17954
-43230 -68421
-791 43138
75245 -66008
-88410 -53431
41182 -46721
85584 77221
6642 44024
-15585 16821
27819 23665
-79039 -7421
3175 13242
-27482 81598
3753 -65901
-86198 -61327
-16102 -47138
-50963 56889
-77683 -65589
31478 -898
56502 14838
-44754 14810
93019 -83375
-66510 -76139
-71483 -40155
-36147 -92657
-26852 89068
-35899 -44125
95235 -45964
-17480 -40006
-15935 -64123
29582 99685
95945 -62531
-15261 83995
37148 -64402
53197 -31420
45232 81547
63128 -33332
54887 44743
-3142 22383
58966 -79979
24474 -43767
60562 -31018
37423 58611
-38495 13734
-81503 -81756
-52593 -89183
-32409 72860
70038 -17412
27516 -66147
-25832 20247
59250 83985
-45462 -13954
-1824 72592
45662 -4891
-42598 82883
83169 -99909
30179 -88797
-14218 -12
-95493 -2877
94617 94311
-44785 43513
26985 -46084
-12286 27928
-50561 -71965
-86144 59799
-6579 -95844
42769 13168
-73264 34801
82439 -45201
1638 -95533
-57552 64570
56237 -10481
-73887 75682
41963 -513
63040 51099
-87659 -86428
7939 -65225
40808 29488
91916 39151
-84577 -97804
-9275 -16070
40126 -42961
83377 17154
36069 -73908
-76930 -35558
12030 86741
-29852 79595
-52052 95468
41029 7834
-33970 29524
59242 61314
85599 -67363
29060 38128
63530 96481
-58852 97686
69933 37848
18858 -10665
-7238 -28264
78608 40433
-17803 -88675
16810 44120
71461 -18314
57243 83801
31884 31734
-1078 -62634
-99457 -46197
-67686 93784
88797 -37919
-83180 39005
-59678 -669
-92945 -73028
-97374 33857
-42335 -37204
-17541 -36812
-10467 -54742
96208 -6676
-31598 45549
95102 11974
39836 -19910
-71820 35465
39764 -90656
8557 -47670
6057 -14363
-2099 -63072
46216 94228
-16695 -25880
85672 6238
78251 -66858
35340 -88500
-63598 -72835
-7998 -20981
34574 -55322
23514 85557
74573 4937
10955 -80517
32222 90338
58575 -63553
-37563 90603
81051 994
78735 -9455
-6231 -51111
-67195 -63950
-98740 -82963
53826 -70155
-73902 -53104
8305 -26835
23701 -11919
862 91151
-22631 -96142
-84982 -20832
-92939 8348
51299 -17183
-74919 -29574
33778 11219
-51430 -65975
83897 25022
-13099 481
-91070 88045
-14660 -10109
-84143 -41271
42800 70288
-84177 -71232
-37739 10657
-56811 45419
-79864 -17486
-22680 -17996
-68532 -33067
-1575 75130
97999 5579
-61560 88838
25960 94433
-62269 -98982
-64760 63394
-41482 33830
2939 -53288
-14837 -29942
-85926 50735
-99250 -18964
14325 -36185
9624 -17117
-16072 -40621
-80348 -78055
-50908 -52097
76631 79225
-28206 84757
-95967 39763
-68494 -57435
13940 -82443
33647 69793
65914 25930
8757 -94488
93325 -56295
82527 -73642
31368 -89278
-39235 22566
38629 -9738
-27582 -15926
-95270 -86286
54304 -77756
-99945 74906
-44624 -35067
25466 22620
58685 -52409
-50593 -84165
-66541 59694
51418 -68097
25505 -32128
36251 28956
-84897 2882
91707 -52883
40270 -5267
6648 30468
43812 -93092
-83438 -85493
-32036 30022
-63699 56374
-7559 -94372
-56614 17894
80504 38332
-32795 -14446
-81152 24422
43592 28550
-51113 28844
51411 -51090
29124 80588
32484 -50035
11618 -82528
55973 36205
-51028 32056
-93826 53220
71222 -72280
-82615 -69691
33824 69511
-43653 -9567
89071 -81310
23251 -66370
-68036 -24790
-45520 57574
-85005 -60318
64626 54483
5471 -18861
-48391 -15680
-7031 -6017
2429 76894
90517 -54598
-58279 75279
-36190 82156
-35127 92592
-44921 40
27430 40902
-47751 -48355
-62359 27699
34118 -39611
-76400 -32005
9617 83214
66696 67054
-95178 -58344
-56239 -24403
7576 -6867
44861 -24229
-17041 -22789
52419 -96583
-60934 -31715
67120 72682
12478 -66579
54773 30473
36119 78414
99017 -40520
6650 1400
93753 18073
57849 -42026
-68351 20070
-64150 90372
10073 48227
94543 -91882
32402 54102
-48153 -679
70971 -97209
49453 -82779
30500 64459
-31224 89496
-17601 356
93868 81516
63714 -60483
-53263 34566
70952 -9985
19241 -98142
-95082 -54656
48088 65412
-55626 43463
3833 -88048
63805 17838
30713 -39248
-93482 -6785
43960 88098
-16713 -6812
85699 69196
86009 -512
-68358 20020
63994 -66903
-98723 60613
88169 655
