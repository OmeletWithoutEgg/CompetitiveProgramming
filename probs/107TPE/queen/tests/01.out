1118.00 28292.00
